ingest/src/main/java/com/microsoft/azure/kusto/ingest/ManagedStreamingIngestClient.java

```diff
@@ -39,19 +37,21 @@
  * Since the streaming client communicates directly with the engine, it's more prone to failure, so this class
  * holds both a streaming client and a queued client.
  * It tries {@value ATTEMPT_COUNT} times using the streaming client, after which it falls back to the queued streaming client in case of failure.
- * If the size of the stream is bigger than {@value MAX_STREAMING_SIZE_BYTES}, it will fall back to the queued streaming client.
+ * By default the policy for choosing a queued ingestion on the first try is the checking of weather the size of the estimated
+ * raw stream size (a conversion to compressed CSV) is bigger than 4MB, it will fall back to the queued streaming client.
+ * Use SourceInfo.size to override size estimations, alternatively - use setQueuingPolicy to override the predicate heuristics.
```

Comment:
SourceInfo.size doesn't exist (as far as I can tell), we have rawdatasize, but I think this passage implies you can lie in it to sneak in bigger files, which I don't like

Reply:
1. the name is still wrong (it's SourceInfo.ge/settRawSizeInBytes)
2. I think just putting it right next to the previous line with this wording is confusing, since it makes it seem that they are related.